src/analysis/imports_sortedness.d

```diff
@@ -66,7 +66,7 @@ class ImportSortednessCheck : BaseAnalyzer
 
 private:
 
-	int level = 0;
```

Comment:
@BBasile I'll try to review your PR, but I don't have much knowledge of D-Scanner's internals.